app/assets/javascripts/components/common/feedback.jsx

```diff
@@ -37,16 +44,32 @@ const Feedback = React.createClass({
     this.hide();
   },
 
+  handleSubmit(event) {
+    const subject = `/revision_feedback?title=${this.titleParam()}`;
+    const body = event.target.value;
+    this.setState({ feedbackSent: 'sending' });
+    API.postFeedbackFormResponse(subject, body).then(() => {
+      this.setState({ feedbackSent: true });
+    });
+    event.preventDefault();
+  },
+
```

Comment:
The submit button posts to Rails API to create Feedback Form Response. This approach doesn't check if the request was successful just that it receives a response. I couldn't figure out a way to figure out if the response was successful. The data received on success is a HTML page.